_data/examples.json

```diff
@@ -161,7 +161,8 @@
     },
     {
       "name": "layer_bar_annotations",
-      "title": "Bar chart with highlighting annotation overlay"
+      "title": "Bar chart that highlights values beyond a threshold",
+      "description": "A bar chart encodes quantitative values, that marks a user specified data domain value, and highlights the area above this value."
```

Comment:
Description could be multiple sentences.  (Think about it as if you have to teach someone briefly about tricks you use in this visualization, how would you explain it to other people.)

Something like: 

To create a bar chart that highlights values beyond a threshold, we use two `layer`s of `bar` marks. The lower layer shows all the bars while the upper layer shows data with values above the threshold in red (#...colorno..).  We then `layer` a `rule` and a `text` mark to annotate the threshold on top of the bars.